ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh

```diff
@@ -33,8 +33,46 @@ pushd ~/tempest/openshift
 
 discover-tempest-config --os-cloud ${OS_CLOUD} --debug --create
 
+# Generate skiplist and allow list
+ORG="openstack-k8s-operators"
+# Check org and project from job's spec
+REF_REPO=$(echo ${JOB_SPEC} | jq -r '.refs.repo')
+REF_ORG=$(echo ${JOB_SPEC} | jq -r '.refs.org')
+
+# Fails if step is not being used on openstack-k8s-operators repos
+# Gets base repo name
+BASE_OP=${REF_REPO}
+if [[ "$REF_ORG" != "$ORG" ]]; then
+    echo "Not a ${ORG} job. Checking if isn't a rehearsal job..."
+    EXTRA_REF_REPO=$(echo ${JOB_SPEC} | jq -r '.extra_refs[0].repo')
+    EXTRA_REF_ORG=$(echo ${JOB_SPEC} | jq -r '.extra_refs[0].org')
+    #EXTRA_REF_BASE_REF=$(echo ${JOB_SPEC} | jq -r '.extra_refs[0].base_ref')
+    if [[ "$EXTRA_REF_ORG" != "$ORG" ]]; then
+      echo "Failing since this step supports only ${ORG} changes."
+      exit 1
+    fi
+    BASE_OP=${EXTRA_REF_REPO}
+fi
+
+
 set +e
-tempest run --regex 'tempest.api.identity.*.v3'
+
+if [ "${TEMPEST_REGEX:-}" ]; then
```

Comment:
Can we add this TEMPEST_REGEX as test env to see if it will work?

https://github.com/openshift/release/blob/master/ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml#L115-L125